src/dcd_mapping/annotate.py

```diff
@@ -53,6 +54,27 @@ def _get_vrs_1_3_ext(allele: Allele) -> Extension:
     )
 
 
+def _get_va_digest(allele: Allele) -> Extension:
+    """Return the VA digest for a pre-mapped allele
+    :param allele: A pre-mapped variant
+    :return A VRS extension reporting the pre-mapped digest
+    """
+    return Extension(name="pre_mapped_digest", value=allele.id)
```

Comment:
should we call this `pre_mapped_id` since in this form it'll include more than just the digest? or on the other hand, strip the ID stuff from the value?

Reply:
I agree, I think it should be renamed to `pre_mapped_id`